### PR TITLE
zjut-LDFnet

### PR DESCRIPTION
**Pull request by ClimBin**
submitted by guyubin from zjut

### CODE REVIEW

File: LDFnet/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Split the ground truth into body map and detail map, which will be saved into `data/DUTS-TR/body-origin` and `data/DUTS-TR/detail-origin`
+```shell
+    python3 utils.py
+```
+- Train the model and get the predicted maps, which will be saved into `eval/DUTS-TR`
+```shell
+    cd model
+    python3 train.py
+    python3 test.py
+```
```
Comment (Ldpe2G) [inline LDFnet/README.md:36]: 这里指示有点迷惑，我看还提供了train.sh 脚本，这里没提到。还有上面预训练模型怎么用，没有提供脚本

**Comment by Ldpe2G** [comment]
建议再仔细对照 [模型验收&pr提交标准](https://github.com/Oneflow-Inc/models/issues/38)

File: LDFnet/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+pip3 install -r requirements.txt
+```
+
+## Data preparation
+Download the dataset and unzip them into according data folder.
+
+-[DUTS-TR](http://saliencydetection.net/duts/download/DUTS-TR.zip) / 
+-[DUTS-TE](http://saliencydetection.net/duts/download/DUTS-TE.zip)
+
```
Comment (Ldpe2G) [inline LDFnet/README.md:16]: 数据集的下载直接加到 train.sh 脚本中

File: LDFnet/model/Resnet50.py [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+            if isinstance(m, nn.Conv2d):
+                n = m.kernel_size[0] * m.kernel_size[1] * m.out_channels
+                m.weight.data.normal_(0, math.sqrt(2.0 / n))
+            elif isinstance(m, nn.BatchNorm2d):
+                m.weight.data.fill_(1)
+                m.bias.data.zero_()
+
+    def initialize(self):
+        self.load_param('pretrained/resnet50-19c8e357')
+        # self.load_state_dict(flow.load('pretrained/resnet50_pretrained_model'), strict=False)
```
Comment (Ldpe2G) [inline LDFnet/model/Resnet50.py:285]: 删除

File: LDFnet/model/Resnet50.py [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+        for m in self.modules():
+            if isinstance(m, nn.Conv2d):
+                n = m.kernel_size[0] * m.kernel_size[1] * m.out_channels
+                m.weight.data.normal_(0, math.sqrt(2.0 / n))
+            elif isinstance(m, nn.BatchNorm2d):
+                m.weight.data.fill_(1)
+                m.bias.data.zero_()
+
+    def initialize(self):
+        self.load_param('pretrained/resnet50-19c8e357')
```
Comment (Ldpe2G) [inline LDFnet/model/Resnet50.py:284]: 模型路劲作为参数传入

File: LDFnet/model/test.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        with flow.no_grad():
+            for i in range(len(self.data.samples)):
+                image, shape, name = self.data[i]
+                image = image[np.newaxis, :]
+                image = flow.Tensor(image).to('cuda')
+
+                outb1, outd1, out1, outb2, outd2, out2 = self.net(image, shape)
+                out = out2
+                pred = flow.sigmoid(out[0, 0]).to('cpu').numpy() * 255
+                head = '../eval/maps/' + self.cfg.datapath.split('/')[-1]
```
Comment (Ldpe2G) [inline LDFnet/model/test.py:43]: 路径作为参数，从脚本传入

File: LDFnet/README.md (hunk 1 above)
Comment (ClimBin) [reply LDFnet/README.md:36]: resnet预训练模型用于train，放入/pretrained文件夹。

Comment (ClimBin) [reply LDFnet/README.md:36]: LDFmodel放入\out文件夹，用于test.py
